tools/pipelines/build-client.yml

```diff
@@ -188,7 +188,7 @@ extends:
     interdependencyRange: ${{ parameters.interdependencyRange }}
     packageManagerInstallCommand: 'pnpm i --frozen-lockfile'
     packageManager: pnpm
-    testCoverage: true
+    testCoverage: ${{ eq(variables['System.TeamProject'], 'public' ) }} # disabling code coverage for internal project since we don't use it
```

Comment:
Do we use it for the public pipeline? Don't we have a separate pipeline for code coverage?

Reply:
The separate pipeline only runs against commits in main to generate the baselines. For PRs we only run build-client so this ensures PR builds have their own coverage measurements, which then get compared against the baseline.